src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <p className="Person__age">{`${person.age === undefined ? '' : ` I am ${person.age}`}`}</p>
```

Comment:
The condition for displaying age is not handling the case when age is null. Please add a check for null values.